screens/ChangePassword.screen.jsx

```diff
@@ -6,83 +6,62 @@ import colors from 'themes/colors';
 import locales from 'constants/locales';
 import placements from 'themes/placements';
 import fonts from 'themes/fonts';
-import { FHeading } from 'components/Composition/FHeading';
 import inputTypes from 'constants/inputTypes';
 import { FInput } from 'components/Inputs/FInput';
 import buttonTypes from 'constants/buttonTypes';
 import { FButton } from 'components/Buttons/FButton';
-import { View } from 'react-native';
+import { View, StyleSheet } from 'react-native';
 import { FImage } from 'components/Composition/FImage';
 import images from 'constants/images';
+import { FKeyboardWrapper } from '../components/Utils/FKeyboardWrapper';
 
 export const ChangePasswordScreen = () => (
-  <FDefaultLayout
-    hasFlatList={false}
-    withLogo={false}
-  >
-    <View style={styles.imageContainer}>
-      <FImage
-        imagePath={images.CHANGE_PASSWORD()}
-        width={sizes.WIDTH_120}
-        height={sizes.HEIGHT_120}
+  <FKeyboardWrapper>
```

Comment:
Musisz zamienić kolejność, Keyboard wrappera dajemy do layoutu, Będzie mówił że ma problem wiec to co dajesz do wrappera musisz w tym przypadku wsadzić w <></>